pkg/cli/mode/mode.go

```diff
@@ -4,18 +4,22 @@ package mode
 
 import "src.elv.sh/pkg/ui"
 
-// ModeLine returns a text styled as a modeline.
-func ModeLine(content string, space bool) ui.Text {
+// modeLine returns a text styled as a modeline.
```

Comment:
godoc for unexported symbols should just elide the symbol:

> // Returns a ...

Reply:
I didn't know that. Done.

Reply:
There isn't a strong consensus in the Go community, some people prefer writing out the symbol (in lower case), but I've settled on this pattern, since this avoids starting an English sentences with a lower-case letter. I'm probably influenced by Google's [C++ style guides](https://google.github.io/styleguide/cppguide.html#Function_Comments) on this matter.